Tests/Unit/EmogrifierTest.php

```diff
@@ -762,6 +768,42 @@ public function emogrifyFormatsCssDeclarations($cssDeclarationBlock, $expectedSt
     }
 
     /**
+     * Data provider for emogrifyInvalidDeclaration.
+     *
+     * @return string[][]
+     */
+    public function invalidDeclarationDataProvider()
+    {
+        return [
+            'missing dash in property name' => ['font weight: bold;'],
+            'invalid character in property name' => ['-9webkit-text-size-adjust:none;'],
+            'missing :' => ['-webkit-text-size-adjust none'],
+            'missing value' => ['-webkit-text-size-adjust :'],
+        ];
+    }
+
+    /**
+     * @test
+     *
+     * @param string $cssDeclarationBlock the CSS declaration block (without the curly braces)
```

Comment:
For the parameter, either it should be plural (declarationsBlock) or just "declaration" (as we always have only one declaration for this test case). (In MDN (which I use as reference for this), they use the plural term: https://developer.mozilla.org/de/docs/Web/CSS/Syntax)